src/main/rx/crsf.c

```diff
@@ -146,13 +145,12 @@ typedef struct crsfPayloadLinkstatistics_s {
     uint8_t Downlink_RSSI;
     uint8_t Downlink_Link_quality;
     uint8_t Downlink_SNR;
+}crsfLinkStatistics_t;
 
-}crsfPayloadLinkstatistics_t;
-
-static crsfPayloadLinkstatistics_t crsfRXLinkstatistics;
+//static crsfPayloadLinkstatistics_t crsfRXLinkstatistics;
 static uint32_t LastLinkStatisticsFrameUs;
 
-int converdbm( int x){
+static uint8_t crsfConvertRssidBm(int8_t x) {
     x = x * -1;
```

Comment:
nitpick alert
`x *= -1`

Reply:
remove need to convert from -1
static uint8_t crsfConvertRssidBm(uint8_t x)
{
    if (x > 92){
      return 1;
    }
    if (x < 21){
      return 100;
    }
    return (uint8_t)(((-0.0154*x*x)-(-0.3794*x)+98.182) + 0.5f);
}